src/app/components/NavBar.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import BottomNavigation from "@mui/material/BottomNavigation";
+import { BottomNavigationAction } from "@mui/material";
+import { SvgIconComponent } from "@mui/icons-material";
+
+interface INavBarItem {
+  label: string;
+  icon: React.ReactElement<SvgIconComponent>;
+  href: string;
+  // active: boolean;
```

Comment:
nit: do we need this + what is active for?

Reply:
I was thinking to display a different (active) icon when the user is on that page/tab, but we can handle that next time.